src/certificates/certificates-card/certificate-signatories/signatory/Signatory.test.jsx

```diff
@@ -71,4 +71,15 @@ describe('Signatory Component', () => {
 
     expect(getByRole('presentation')).toBeInTheDocument();
   });
+
+  it('shows confirm modal on delete icon click', async () => {
+    const { getByLabelText, getByText } = renderSignatory(
+      { ...defaultProps, componentMode: MODE_STATES.create, showDeleteButton: true },
+    );
+    const deleteIcon = getByLabelText(messages.deleteTooltip.defaultMessage);
+
+    userEvent.click(deleteIcon);
+
+    expect(getByText(`Delete "${defaultProps.name}" from the list of signatories?`)).toBeInTheDocument();
```

Comment:
[important]: Please use an [existing](https://github.com/raccoongang/frontend-app-course-authoring/blob/6ef7e35e260f2274f878b1793aab1a78832ebdb2/src/certificates/certificates-card/messages.js#L126) translation. For [example](https://github.com/raccoongang/frontend-app-course-authoring/blob/6ef7e35e260f2274f878b1793aab1a78832ebdb2/src/course-team/info-modal/InfoModal.test.jsx#L63)

Reply:
You have already added a check for this text below. Do we need the legacy version?